pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -373,6 +374,10 @@ func (e *cloudWatchExecutor) QueryData(ctx context.Context, req *backend.QueryDa
 		result, err = e.executeTimeSeriesQuery(ctx, req)
 	}
 
+	if err != nil {
+		metrics.MCloudwatchQueriesFailuresTotal.WithLabelValues(model.QueryType).Inc()
```

Comment:
I would not recommend to allow any value of `model.QueryType` here since it can in theory be used to overload a Prometheus instance ([high cardinality](https://prometheus.io/docs/practices/naming/#labels)) and/or allow XSS content. Prometheus labels also only allow certain characters (can't tell what they are currently) so whenever you add a label coming from user input you should sanitize it.